src/Discord/Parts/User/User.php

```diff
@@ -72,7 +71,7 @@ public function sendMessage($message, $tts = false, $embed = null)
         $deferred = new Deferred();
 
         $this->getPrivateChannel()->then(function ($channel) use ($message, $tts, $embed, $deferred) {
-            $channel->sendMessage($message, $tts, $embed)->then(function ($response) use ($deferred) {
+            $channel->sendMessage($message, $tts, $embed)->then(function ($response) use ($channel, $deferred) {
```

Comment:
Nope, it's left over from when I was storing the message after creating it (I didn't know at the time about a MESSAGE_CREATE event being fired after creating the message lol